src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -130,6 +133,8 @@
     private int messagesInTheSystem = 0;
 
     private Statistics statistics = new Statistics();
+    
+    private static IndexTimestamp indexTime = new IndexTimestamp();
```

Comment:
the truth is, that every project has its own index, so this should be a per project variable, but this is a good step forward, so let's start with this